lib/util/yaml.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  DumpOptions,
+  LoadOptions,
+  loadAll as multiple,
+  load as single,
+  dump as upstreamDump,
+} from 'js-yaml';
+import { regEx } from './regex';
+
+interface YamlOptions extends LoadOptions {
+  removeTemplates?: boolean;
+}
+
+export function loadAll(
+  content: string,
+  iterator?: null | undefined,
+  options?: YamlOptions,
+): unknown[] {
+  const massagedContent = massageContent(content, options);
+
+  return multiple(massagedContent, iterator, options);
+}
+
+export function load(content: string, options?: YamlOptions): unknown {
+  const massagedContent = massageContent(content, options);
+  return single(massagedContent, options);
+}
+
+export function dump(obj: any, opts?: DumpOptions | undefined): string {
+  return upstreamDump(obj, opts);
+}
+
+export function massageContent(content: string, options?: YamlOptions): string {
```

Comment:
do we really need to export this? it's already called by load.

Reply:
I have planned to write separate tests for it, but haven't done that in the end.